app/src/forms/form/service.js

```diff
@@ -685,15 +667,7 @@ const service = {
       await FormVersionDraft.query().deleteById(formVersionDraftId);
       await trx.commit();
 
-      const { subscribe } = await service.readForm(formId);
-      if (subscribe && subscribe.enabled) {
-        const subscribeConfig = await service.readFormSubscriptionDetails(formId);
-        const config = Object.assign({}, subscribe, subscribeConfig);
-        const formVersion = new FormVersion();
-        formVersion.id = version.id;
-        formVersion.formId = formId;
-        service.postSubscriptionEvent(config, formVersion, null, SubscriptionEvent.FORM_DRAFT_PUBLISHED);
-      }
+      eventService.publishFormEvent(formId, version.id, version.published);
```

Comment:
I don't see how this is sending the `FORM_DRAFT_PUBLISHED` event? `eventService.publishFormEvent` only sends the `FORM_PUBLISHED` and `FORM_UNPUBLISHED` events.